standalone-metastore/metastore-common/src/gen/thrift/gen-javabean/org/apache/hadoop/hive/metastore/api/AddDynamicPartitions.java

```diff
@@ -787,13 +787,13 @@ public void read(org.apache.thrift.protocol.TProtocol iprot, AddDynamicPartition
           case 5: // PARTITIONNAMES
             if (schemeField.type == org.apache.thrift.protocol.TType.LIST) {
               {
-                org.apache.thrift.protocol.TList _list828 = iprot.readListBegin();
-                struct.partitionnames = new java.util.ArrayList<java.lang.String>(_list828.size);
-                @org.apache.thrift.annotation.Nullable java.lang.String _elem829;
-                for (int _i830 = 0; _i830 < _list828.size; ++_i830)
```

Comment:
I don't think this change should be here, did you generated the thrift files with thrift version 0.13.0? That should be used in upstream

Reply:
I've verified the thrift version is correct and I can compile it correctly on the current master branch. These changes are generated after I introduced three new structs.